rewrite-maven/src/main/java/org/openrewrite/maven/AddParentPom.java

```diff
@@ -151,6 +160,7 @@ public Xml.Document visitDocument(Xml.Document document, ExecutionContext ctx) {
             private Optional<String> findAcceptableVersion(String groupId, String artifactId, ExecutionContext ctx)
                     throws MavenDownloadingException {
                 if (availableVersions == null) {
+                    ResolutionEventListener resolutionListener = new MavenExecutionContextView(ctx).getResolutionListener();
```

Comment:
this seems to be unused

Reply:
Ah good catch, thank you! That was a holdover from a previous commit.